houdini/property_contexts

```diff
@@ -1,2 +1,2 @@
-ro.enable.native.bridge.    u:object_r:vendor_houdini_prop:s0
-ro.dalvik.vm.isa.   u:object_r:exported_dalvik_prop:s0
+ro.vendor.enable.native.bridge.exec    u:object_r:vendor_houdini_default_prop:s0
+ro.vendor.enable.native.bridge.exec64    u:object_r:vendor_houdini_default_prop:s0
```

Comment:
You can just use one line
ro.vendor.enable.native.bridge.    u:object_r:vendor_houdini_default_prop:s0
to instead these two lines.